tensorflow/core/kernels/mkl/mkl_concat_op.cc

```diff
@@ -364,7 +364,7 @@ class MklConcatFwdPrimitive : public MklPrimitive {
              const std::vector<memory::desc>& srcs_md) {
     // Create memory descriptors for concat with specified srcs format
     for (size_t i = 0; i < concat_fwd_dims.num_inputs; i++) {
-      dnnl::memory::desc source_md(memory::desc(GET_MEMORY_DESC(srcs_md[i])));
+      dnnl::memory::desc source_md((memory::desc(GET_MEMORY_DESC(srcs_md[i]))));
```

Comment:
What clang error did we get without the extra pair of parentheses? Could you please post the error message? Thank you!

Reply:
@penpornk Below is the error from Clang:
```
tensorflow/core/kernels/mkl/mkl_concat_op.cc:367:35: warning: parentheses were disambiguated as a function declaration [-Wvexing-parse]
      dnnl::memory::desc source_md(memory::desc(GET_MEMORY_DESC(srcs_md[i])));
                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
tensorflow/core/kernels/mkl/mkl_concat_op.cc:367:36: note: add a pair of parentheses to declare a variable
      dnnl::memory::desc source_md(memory::desc(GET_MEMORY_DESC(srcs_md[i])));
                                   ^
                                   (                                        )
tensorflow/core/kernels/mkl/mkl_concat_op.cc:368:23: error: no matching member function for call to 'push_back'
      context_.src_md.push_back(source_md);
      ~~~~~~~~~~~~~~~~^~~~~~~~~
```

Reply:
Got it. Thank you very much!